src/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/StandardCommands.cs

```diff
@@ -4,409 +4,329 @@
-        ///       Gets the GUID/integer value pair for the PropertiesWindow command. Read
-        ///       only.
-        ///    </para>
+        /// Gets the GUID/integer value pair for the PropertiesWindow command. Read only.
         /// </summary>
         public static readonly CommandID PropertiesWindow = new CommandID(s_standardCommandSet, VSStandardCommands.cmdidPropertiesWindow);
+        
         /// <summary>
-        ///    <para>
-        ///       Gets the GUID/integer value pair for the LockControls command. Read only.
-        ///    </para>
+        /// Gets the GUID/integer value pair for the LockControls command. Read only.
         /// </summary>
         public static readonly CommandID LockControls = new CommandID(s_standardCommandSet, VSStandardCommands.cmdidLockControls);
+
         /// <summary>
-        ///    <para>
-        ///       Gets the GUID/integer value pair for the F1Help command. Read only.
-        ///    </para>
+        /// Gets the GUID/integer value pair for the F1Help command. Read only.
         /// </summary>
         public static readonly CommandID F1Help = new CommandID(s_standardCommandSet, VSStandardCommands.cmdidF1Help);
 
         // Component Tray Menu commands...
         /// <summary>
-        ///    <para>
-        ///       Gets the GUID/integer value pair for the ArrangeIcons command. Read only.
-        ///    </para>
+        /// Gets the GUID/integer value pair for the ArrangeIcons command. Read only.
         /// </summary>
         public static readonly CommandID ArrangeIcons = new CommandID(s_ndpCommandSet, cmdidArrangeIcons);
+
         /// <summary>
-        ///    <para>
-        ///       Gets the GUID/integer value pair for the LineupIcons command. Read only.
-        ///    </para>
+        /// Gets the GUID/integer value pair for the LineupIcons command. Read only.
         /// </summary>
         public static readonly CommandID LineupIcons = new CommandID(s_ndpCommandSet, cmdidLineupIcons);
+
         /// <summary>
-        ///    <para>
-        ///       Gets the GUID/integer value pair for the ShowLargeIcons command. Read
-        ///       only.
-        ///    </para>
+        /// Gets the GUID/integer value pair for the ShowLargeIcons command. Read only.
         /// </summary>
         public static readonly CommandID ShowLargeIcons = new CommandID(s_ndpCommandSet, cmdidShowLargeIcons);
 
         /// <summary>
-        ///    <para> Gets the first of a set of verbs.
-        ///       Read only.</para>
+        /// Gets the first of a set of verbs. Read only.
         /// </summary>
-        public static readonly CommandID VerbFirst = new CommandID(s_ndpCommandSet, cmdidDesignerVerbFirst);
+        public static readonly CommandID VerbFirst = new CommandID(s_ndpCommandSet, CmdidDesignerVerbFirst);
 
         /// <summary>
-        ///    <para> Gets the last of a set of verbs.
-        ///       Read only.</para>
+        ///  Gets the last of a set of verbs.
```

Comment:
Nit: extra space at the beginning of the text

Reply:
Fixed